x-pack/plugins/security_solution/public/entity_analytics/components/entity_details_flyout/tabs/risk_inputs.tsx

```diff
@@ -30,6 +40,90 @@ export interface AlertRawData {
   _id: string;
 }
 
+const FieldLabel = styled.span`
+  font-weight: ${euiLightVars.euiFontWeightMedium};
+  color: ${euiLightVars.euiTitleColor};
+`;
+
+const CriticalityField: React.FC<{ criticalityLevel?: CriticalityLevel }> = ({
```

Comment:
Maybe this can/should also be in the `asset_criticality` folder?

Reply:
Done. Moved to a common component as something almost identical is used in the selector 👍 

See https://github.com/elastic/kibana/pull/174638/commits/4b2ad172d30cde80deb679dcd9046118e7da1d8d

Reply:
Nice 🥳